### DISCUSSION TITLE
Which branch/tag should I use to build with qtermwidgets dependency

### DISCUSSION THREAD

**Discussion by yscontrol** [lxqt/lxqt#2052] [General]
Which branch/tag should git checkout to build tool for building qtermwidget https://github.com/lxqt/qtermwidget with Qt 5.12.10 on Ubuntu 16.04 ? I see latest commit dropped support for 5.12 and  "The latest lxqt-build-tools is also needed for compilation." is indicated  for qtermwidget installation

**Comment by tsujan** [comment]
I if correctly remember, Qt 5.12 was supported by LXQt 0.17 (the latest release until now) but, because Ubuntu 16.04 is too old, you'll be alone with the problems you might encounter.
